pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -835,6 +836,72 @@ func TestUpdateTaskRunsState(t *testing.T) {
 
 }
 
+func TestUpdateTaskRunsWithWhenExpressions(t *testing.T) {
+	// TestUpdateTaskRunsWithWhenExpressions runs "getTaskRunsStatus" and verifies how it updates a PipelineRun status
+	// from a TaskRun with When Expressionsassociated to the PipelineRun
```

Comment:
i think this would make sense as a docstring on the function? or maybe this is a convention other tests have used? kinda confuses me tho, doesn't seem like the right style for go

Reply:
yes, the other tests for pipelineruns have the docstrings -- can exclude it for this test case